misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractCharSequenceToContainAtMostExpectationsTest.kt

```diff
@@ -0,0 +1,234 @@
+package ch.tutteli.atrium.specs.integration
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.*
+import ch.tutteli.atrium.testfactories.TestFactory
+import ch.tutteli.atrium.translations.DescriptionCharSequenceExpectation
+import ch.tutteli.atrium.specs.integration.CharSequenceToContainSpecBase.Companion.text
+import ch.tutteli.atrium.specs.integration.CharSequenceToContainSpecBase.Companion.helloWorld
+import ch.tutteli.atrium.specs.integration.CharSequenceToContainSpecBase.Companion.toContainIgnoringCase
+
+abstract class AbstractCharSequenceToContainAtMostExpectationsTest(
```

Comment:
Can you please also add
```suggestion
@Suppress("FunctionName")
abstract class AbstractCharSequenceToContainAtMostExpectationsTest(
```